src/cli.rs

```diff
@@ -822,6 +824,12 @@ impl BirliContext {
         if matches.is_present("flag-autos") {
             flag_ctx.autos = true;
         }
+        if matches.is_present("flag-dc") {
+            flag_ctx.flag_dc = true;
```

Comment:
All we need to get this PR over the line is to get the diff coverage up. a simple test in `mod argparse_tests` that reads an mwax obs (e.g. `1297526432` ) with `--flag-dc` and checks that `flag_ctx.flag_dc` is true should  help!